samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -175,11 +175,29 @@ int main(void)
 		printk("BIG create complete chan %u.\n", chan);
 	}
 
+	/* Simulate delayed streaming on one stream */
+	const uint8_t skip_chan = UINT8_MAX;
+	uint32_t skip = 5U;
```

Comment:
I would be happy if we could remove testing code out from the sample. If we want to document how to do these kinds of things we could consider documenting it in the readme.

Reply:
Sure will remove this commit that adds the check in skipped ISO data send.